src/aiida/cmdline/commands/cmd_presto.py

```diff
@@ -0,0 +1,133 @@
+    control, please use the command `verdi profile setup` instead.
+
+    The main goal for this command is to setup a profile with minimal requirements to make it easy to install AiiDA and
+    get started as quickly as possible. Therefore, by default, the created profile uses the `core.sqlite_dos` storage
+    plugin which does not require any services, such as PostgreSQL and RabbitMQ are not required. This _does_ mean,
+    however, that not all functionality of AiiDA is available, most notably running the daemon and submitting processes
+    to said daemon.
+
+    The command performs the following actions:
+
+    \b
+    * Create a new profile that is set as the new default
+    * Create a default user for the profile (email can be configured through options)
+    * Set up the localhost as a `Computer` and configure it
+    * Set a number of configuration options with sensible defaults
+
+    """
+    from aiida.brokers.rabbitmq.defaults import detect_rabbitmq_config
+    from aiida.common import exceptions
+    from aiida.manage.configuration import create_profile, load_profile
+    from aiida.orm import Computer
+
+    storage_config: dict[str, t.Any] = {}
+    storage_backend = 'core.sqlite_dos'
+
+    broker_config = detect_rabbitmq_config()
+    broker_backend = 'core.rabbitmq' if broker_config is not None else None
+
+    if broker_config is None:
+        echo.echo_report('RabbitMQ server not found: configuring the profile without a broker.')
+    else:
+        echo.echo_report('RabbitMQ server detected: configuring the profile with a broker.')
+
+    try:
+        profile = create_profile(
+            ctx.obj.config,
+            name=profile_name,
+            email=email,
+            storage_backend=storage_backend,
+            storage_config=storage_config,
+            broker_backend=broker_backend,
+            broker_config=broker_config,
+        )
+    except (ValueError, TypeError, exceptions.EntryPointError, exceptions.StorageMigrationError) as exception:
+        echo.echo_critical(str(exception))
+
+    echo.echo_success(f'Created new profile `{profile.name}`.')
+
+    ctx.obj.config.set_option('runner.poll.interval', 1, scope=profile.name)
+    ctx.obj.config.set_default_profile(profile.name, overwrite=True)
+    ctx.obj.config.store()
+
+    echo.echo_info(f'Loaded newly created profile `{profile.name}`.')
+    load_profile(profile.name, allow_switch=True)
+
+    filepath_scratch = pathlib.Path(ctx.obj.config.dirpath) / 'scratch' / profile.name
+
+    computer = Computer(
+        label='localhost',
+        hostname='localhost',
+        description='Localhost automatically created by `verdi presto`',
+        transport_type='core.local',
+        scheduler_type='core.direct',
+        workdir=str(filepath_scratch),
+    ).store()
+    computer.configure(safe_interval=0)
+    computer.set_minimum_job_poll_interval(1)
```

Comment:
I am always confused about this, what is the difference between safe_interval and minimum_job_poll_interval?

Reply:
safe_interval is used by the transport to determine how often a new connection can be made to the computer. The `minimum_job_poll_interval` is used by the engine to determine how often a request can be made to a scheduler to check with the status of jobs.

Reply:
Thanks. So for `core.local` transport, is it always okay to set `safe_interval=0`? Or can it sometimes lead to problems?

Also, does it ever make sense to have poll interval < safe_interval?

Sorry for off-topic. I'll open a separate issue if there is anything that could be improved in the docs. 
(I know it's there, but the relationship between them is AFAIK not fully explained)